mappings/net/minecraft/entity/mob/EndermanEntity.mapping

```diff
@@ -5,7 +5,9 @@ CLASS net/minecraft/unmapped/C_xstqqjuq net/minecraft/entity/mob/EndermanEntity
 	FIELD f_eijumrmj ANGER_TIME_RANGE Lnet/minecraft/unmapped/C_ivtmlivh;
 	FIELD f_febytfzn ATTACKING_SPEED_BOOST Lnet/minecraft/unmapped/C_msqsqbdh;
 	FIELD f_grcebcsx ageWhenTargetSet I
+	FIELD f_hesbeqpf CREEPY_STARE_SOUND_DELAY I
 	FIELD f_hyubxisd lastAngrySoundAge I
+	FIELD f_kwifbmcs MIN_DEAGRESSION_TIME I
```

Comment:
Is `DEAGRESSION` a word? Could maybe me `MIN_AGRESSION_LOSS_TIME`

Reply:
`MIN_ANGER_LOSS_TIME`?

Reply:
`MIN_ANGER_LOSS_TIME` sounds good to me